packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -90,6 +90,7 @@ function fr(key: Key, children: VNodes, stable: 0 | 1): VFragment {
         children: [t(''), ...children, t('')],
         stable,
         owner: getVMBeingRendered()!,
+        trailingAnchor: undefined,
```

Comment:
It seems like we should be able to set the `trailingAnchor` immediately here, since we already have the `t('')` above.

Ditto for `elm` since we already know what the leading anchor is too.